src/site/content/en/lighthouse-accessibility/aria-name/index.md

```diff
@@ -0,0 +1,150 @@
+---
+layout: post
+title: "ARIA items do not have accessible names"
+description: |
+  Learn how to improve your web page's accessibility by making sure that
+  assistive technology users can access the names of ARIA items.
+date: 2020-12-03
+web_lighthouse:
+  - aria-command-name
+  - aria-input-field-name
+  - aria-meter-name
+  - aria-progressbar-name
+  - aria-toggle-field-name
+  - aria-tooltip-name
+  - aria-treeitem-name
+---
+
+{% include 'content/lighthouse-accessibility/about-aria.njk' %}
+
+{% include 'content/lighthouse-accessibility/accessible-names.njk' %}
+
+## How Lighthouse identifies ARIA items without accessible names
+
+[Lighthouse](https://developers.google.com/web/tools/lighthouse/)
+flags custom ARIA items whose names
+aren't accessible to assistive technologies:
+
+<figure class="w-figure">
+  <img class="w-screenshot" src="aria-toggle-field-name.png"
+    alt="Lighthouse audit showing custom toggle elements without accessible names">
+</figure>
+
+There are 7 audits that check for accessible names, each one covers a different
+set of [ARIA roles](https://www.w3.org/TR/wai-aria-practices-1.1/#aria_ex). Elements
+that have any of the following ARIA roles but don't have accessible names will cause this audit to fail:
+- `aria-command-name` - `button`, `link`, `menuitem`
```

Comment:
```suggestion
that have any of the following ARIA roles but don't have accessible names will cause this audit to fail:

- `aria-command-name` - `button`, `link`, `menuitem`
```
I believe I've seen some markdown renderers fail to see this as a paragraph followed by a list, so I'll add a newline to defend against that